daemon/src/taker_cfd.rs

```diff
@@ -301,6 +312,26 @@ impl Actor {
 
         Ok(())
     }
+
+    // TODO: Duplicated with maker
+    async fn handle_commit(&mut self, msg: Commit) -> Result<()> {
```

Comment:
in the `handle_***` functions, you don't need to pass the msg, but just `order_id` in this case (like in other handlers)